dockerfiles/theia/Dockerfile

```diff
@@ -22,6 +22,8 @@ RUN yarn && \
     cd /home/default/theia && \
     yarn && \
     yarn theia build
+RUN ln -sf /home/default/theia/node_modules /home/theia/node_modules
```

Comment:
could you merge these two run instructions with the previous `RUN` instruction (adding `&&`)